.editorconfig

```diff
@@ -14,7 +14,7 @@ trim_trailing_whitespace = true
 indent_style = tab
 indent_size = 4
 
-[{.jshintrc,*.json,*.yml,.babelrc,.bowerrc,.browserslistrc,.postcssrc,*.scss}]
+[{.jshintrc,*.js,*.json,*.yml,.babelrc,.bowerrc,.browserslistrc,.postcssrc,*.scss}]
```

Comment:
@coreylafferty just a note that it looks like this change is missing from your [new PR](https://github.com/forumone/gesso-wp/pull/270).